src/Maestro/Maestro.Web/Properties/launchSettings.json

```diff
@@ -18,12 +10,21 @@
     },
     "Maestro.Web": {
       "commandName": "Project",
+      "commandLineArgs": "get-buil",
```

Comment:
```suggestion
      "commandLineArgs": "get-build",
```
?

Reply:
Actually maybe this whole file can get reverted?

Reply:
yup, not this was a typo